components/os/component_output.go

```diff
@@ -231,25 +230,25 @@ func init() {
 
 	// Get boot ID (Linux-specific)
 	var err error
-	currentMachineMetadata.BootID, err = pkg_host.GetBootID()
+	currentMachineMetadata.BootID, err = pkghost.GetBootID()
```

Comment:
We should not call these functions. just call those functions which return static variable values.